src/pilot/templates/nic-configs/5_port/nic_environment.yaml

```diff
@@ -28,7 +28,7 @@ parameter_defaults:
   ControllerBond1Interface1: em2
   ControllerBond1Interface2: p3p2
   # The bonding mode to use for controller nodes
```

Comment:
The MTU variables that were added to the NIC configuration files should also be added here.

Reply:
The "nic-environment.yaml" file contains settings that are related to the NIC names and bonding options, where as MTU is related to Networks just like network IP ranges and VLAN IDs, so it should be in "network-environment.yaml" files. Moreover, "nic-environment.yaml" is part of the customizable NIC configuration scheme. Hence, it should only contain variables/parameters that are affecting the overcloud NIC configs only. Its not a good idea to make MTU a part of that as we make changes in the SAH, CEPH dashboard VM and director VM based on these settings. 

Also we have "NeutronGlobalPhysnetMtu: 1500" which is used to set the Global value for neutron to create the overcloud tenant networks. This parameter name should stay as it is and its value comes from the "tenant_network_mtu" in the ini file. So, we cannot have it a part of a file that is entirely customizable. 

MTU setting for floating network is done through the extra config i.e. neutron::plugins::ml2::physical_network_mtus: physext:1500, which we cannot make a part of "nic-environment.yaml". 

So as per our understanding network-environment.yaml is the appropriate place to keep these settings.